connector-api/src/main/java/it/pagopa/pdv/tokenizer/connector/exception/TooManyRequestException.java

```diff
@@ -0,0 +1,5 @@
+package it.pagopa.pdv.tokenizer.connector.exception;
+
+public class TooManyRequestException extends RuntimeException {
+}
```

Comment:
refactor rename `TooManyRequestException` into `TooManyRequestsException` and add the exception cause to constructor like the suggested changes
```suggestion
public class TooManyRequestsException extends RuntimeException {

    public TooManyRequestsException(Throwable cause) {
        super(cause);
    }
}
```